.changeset/shaggy-pots-hear.md

```diff
@@ -11,5 +11,6 @@ The information will include:
 - Version_number
```

Comment:
recognize it's not part of the changes of this PR, but just noticed the underscore -- can we remove as a minor cleanup?

```suggestion
- Version number
```